Gemfile

```diff
@@ -40,7 +40,7 @@ gem 'simplecov', '~> 0.8'
 unless RUBY_VERSION == "1.9.2"
```

Comment:
change this to `unless RUBY_VERSION == "1.9.2" || RUBY_ENGINE == 'java'` to disable this on jruby